control-base/src/main/java/fi/nls/oskari/control/layer/SaveOrganizationHandler.java

```diff
@@ -24,15 +24,15 @@
 
     private static final Logger log = LogFactory.getLogger(SaveOrganizationHandler.class);
 
-    private final LayerGroupService layerGroupService = ServiceFactory.getLayerGroupService();
+    private final DataProviderService dataProviderService = ServiceFactory.getDataProviderService();
 
     @Override
     public void handleAction(ActionParameters params) throws ActionException {
 
         final HttpServletRequest request = params.getRequest();
         try {
             final int groupId = params.getHttpParam(PARAM_ID, -1);
-            final LayerGroup group = new LayerGroup();
+            final DataProvider group = new DataProvider();
```

Comment:
Names could be more in-line. `SaveOrganizationHandler` uses `DataProviderService` to store new `DataProvider group`.